catalog/queries/wikidata-entities-streets.rq

```diff
@@ -46,11 +47,15 @@ WHERE {
         ?item schema:description ?description
         FILTER(LANG(?description) = "nl")
     }
+    OPTIONAL {
+        ?item skos:altLabel ?altLabel
+        FILTER(LANG(?altLabel) = "nl")
+    }
     ?item rdfs:label ?streetName .
     ?administration rdfs:label ?administrationName .
     FILTER LANGMATCHES(LANG(?streetName),"nl")
     FILTER LANGMATCHES(LANG(?administrationName),"nl")
-    FILTER STRSTARTS(LCASE(?streetName),LCASE(?query))
+    FILTER (STRSTARTS(LCASE(?streetName),LCASE(?query))||STRSTARTS(LCASE(?altLabel),LCASE(?query)))
```

Comment:
Out of curiosity: does this always yield the right results? Variable `?streetName` is always set, but `?altLabel` is optional - and may not be bound.

Reply:
The filter is based an OR statement with streetName always set. There will be no results when the query string doesn't match streetName and there is no altLabel or altLabel doesn't match the query string either. This would be the right result. The only question might be what happens when LCASE() is performed on an unbound var. Would this result in an error or just a false condition on at the right part of the OR statement. In that case a BOUND test on altLabel should be added. I haven't seen strange behaviour during testing so far.

Reply:
Thanks! I've done a quick test with an unbound variable in combination with an `LCASE()` - and it works fine (i.e. no errors are thrown).